apps/condo/domains/meter/tasks/sendVerificationDateReminder.js

```diff
@@ -179,30 +179,46 @@ const getOrganizationLang = async (context, id) => {
     return get(COUNTRIES, get(organization, 'country.locale'), DEFAULT_LOCALE)
 }
 
-const generateReminderMessages = async ({ context, reminderWindowSize, reminders }) => {
+const generateReminderMessages = async ({ context, reminders }) => {
     const messages = []
     await Promise.all(reminders.map(async (reminder) => {
         const { meter, residents } = reminder
         const lang = await getOrganizationLang(context, meter.organization.id)
+
+        // get context for specific locale
+        const localeDependedContext = {
```

Comment:
it looks hacky. probably it's better to just pass the locale to send mesage

Reply:
Yes. This is because we are using context locale (a locale extracted from context.req object, the logic in locale.js) in order to get proper translation for MeterResource.name field. And since we are running this code not on the client side - we have no locale set up for that. More over - every resident possibly have his own locale - we have to change context based on resident locale